projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -1628,8 +1171,31 @@
 			} );
 
 			carousel.gallery.innerHTML = '';
+
 			initCarouselSlides( gallery.querySelectorAll( settings.imgSelector ), settings.startIndex );
-			selectSlideAtIndex( settings.startIndex );
+			carousel.overlay.style.display = 'block';
+
+			swiper = new Swiper( '.swiper-container', {
```

Comment:
I guess we could pass in `window.Swiper` to the self-executing anon func to keep the linter quiet.

Reply:
yeh, or Andrew's work on making this es6 modules might be a better way around this - but that could be a second phase.

Reply:
> Andrew's work on making this es6 modules might be a better way around this

I think the es6 modules approach seems promising (and I'm [obviously pretty excited](https://github.com/Automattic/jetpack/pull/20131) about it), but agreed we could totally tackle that in a follow-up if it looks like swiper.js isn't too heavy to pull in directly (especially since we're already doing that for the Slideshow block?).